scaling_performance/scaling_cluster_monitoring.adoc

```diff
@@ -0,0 +1,131 @@
+[[scaling-performance-cluster-monitoring]]
+= Scaling Cluster Monitoring Operator
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+{product-title} exposes metrics that can be collected and stored in back-ends by
+the link:https://github.com/openshift/cluster-monitoring-operator[cluster-monitoring-operator]
+As an {product-title} administrator, you can view system resources, containers and components metrics in
+one dashboard interface (Grafana).
+
+This topic provides information on scaling the cluster monitoring operator.
+
+
+[[cluster-monitoring-recommendations-for-OCP]]
+== Recommendations for {product-title}
+* Use at least 3 infra nodes as best practice (for 2 Prometheus pods and operator pods)
+xref:../admin_guide/manage_nodes.adoc#infrastructure-nodes[infrastructure
+nodes].
+* Use at least 3 OCS nodes (with NVMe drives) link:https://www.redhat.com/de/technologies/cloud-computing/openshift-container-storage[*openshift-container-storage*].
+* Use persistent storage when configuring openshift-monitoring. Set
+`USE_PERSISTENT_STORAGE=true`.
+* use GlusterFS as storage on top of non-volatile memory express (NVMe) drives link:https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_storage_glusterfs[*openshift_storage_glusterfs*].
+
+[[cluster-monitoring-capacity-planning]]
+== Capacity Planning for Cluster Monitoring Operator
+
+Various tests were performed for different scale sizes. The Prometheus database grew, as reflected in the table below.
+
+.Prometheus Database storage requirements based on number of nodes/pods in the cluster
+[options="header"]
+|===
+|Number of Nodes |Number of Pods |Prometheus storage growth per day |Prometheus storage growth per 15 days |RAM Space (per scale size) |Network (per tsdb chunk)
+
+|50
+|1800
+|6.3 GB
+|94 GB
+|6 GIB
+|16 MIB
+
+|100
+|3600
+|13 GB
+|195 GB
+|10 GIB
+|26 MIB
+
+|150
+|5400
+|19 GB
+|283 GB
+|12 GIB
+|36 MIB
+
+|200
+|7200
+|25 GB
+|375 GB
+|14 GIB
+|46 MIB
+|===
+
+In the above calculation, approximately 20 percent of the expected size was added as
+overhead to ensure that the storage requirements do not exceed the calculated value.
+
+The above calculation was developed for the default {product-title} *cluster-monitoring-operator*.
+For higher scale, edit the `prometheus_persistent_volume_size` variable, which defaults to `50Gib`.
+
+[NOTE] the CPU Utilization has minor impact, the ratio is ~1 core out of 40 per 50 nodes and 1800 pods.
```

Comment:
[NOTE]
    ====
    CPU utilization has minor impact. The ratio is about 1 core out of 40, per 50 nodes and 1800 pods.
    ====